kahypar/application/command_line_options.h

```diff
@@ -711,7 +720,7 @@ void parseIniToContext(Context& context, const std::string& ini_filename) {
   po::store(po::parse_config_file(file, ini_line_options, true), cmd_vm);
   po::notify(cmd_vm);
 
-  if (context.partition.use_individual_part_weights) {
+  if (context.partition.use_individual_part_weights) {  // Note(Lars): This affects flow network sizes!
     context.partition.epsilon = 0;
```

Comment:
Still up to date. The default config (not eco) uses epsilon to scale up the flow networks.
Flow-based refinement still works fine.
It just doesn't scale the flow problems up as much, as with non-individual part weights.
I'll take another look. The issue I remember is that we don't have perfect balance part weights.
Any thoughts on an appropriate scaling function?

Reply:
I pushed another commit addressing this. If you're happy with it, you can remove the note.
It works as follows.
1) Recompute "a guess" for an epsilon = (sum of max part weights)/(total weight) -1
i.e. what it would be with non-individual part weights
2) for the flow problem scaling we take 1 + (alpha - 1) * adjusted_eps instead of 1 + alpha*eps, because context.partition.perfect_balance_part_weights was previously set to context.partition.max_part_weights